R/zzz.R

```diff
@@ -1,4 +1,7 @@
 .onLoad <- function(lib, pkg) { # nocov start
+  # we need a way to tell httr that a Google Colab session is interactive
+  utils::assignInNamespace("is_interactive", rlang::is_interactive, ns = "httr")
+
```

Comment:
This can go away if https://github.com/r-lib/httr/pull/734 is merged and released.